Fixing formatting in EBS API Section (#134)
C2Devel/terraform-provider-croccloud: docs/data-sources/ebs_volume.md

layout: "aws"
page_title: "aws_ebs_volume"
description: |-

>   Provides information about the volume.

Reviewer: Тут вот `the volume`, а в датасорсе для снапшота `a snapshot`

Reviewer 2: я бы оставил неопределенный артикль, поскольку датасорс описывает тип, а не конкретный ресурс. И здесь первое упоминание

layout: "aws"
page_title: "aws_ebs_volume"
description: |-
  Provides information about a volume.
---

[describe-volumes]: https://docs.k2.cloud/en/api/ec2/volumes/DescribeVolumes.html

# Data Source: aws_ebs_volume

> Provides information about the volume.

Reviewer 2: ```suggestion
Provides information about a volume.
```
Эта строка должна повторять description.
(описание скрыто, диск упоминается первый раз, поэтому неопределенный артикль)

> Provides information about the volume.

Reviewer 2: аналогично

* `filter` - (Optional) One or more name/value pairs to use as filters.
    * _Valid values:_ See supported names and values in [EC2 API documentation][describe-volumes]
* `most_recent` - (Optional) (Bool) If more than one result is returned, use the most recent volume.
    * _Default value:_ `false` 

## Attribute Reference

### Supported attributes

In addition to all arguments above, the following attributes are exported:

* `arn` - (String) The Amazon Resource Name (ARN) of the volume.
* `availability_zone` - (String) The availability zone where the volume will be located.
* `id` - (String) The ID of the volume.
    * _Example:_ `vol-12345678`
* `iops` - (Int) The amount of IOPS for the volume.
* `size` - (Int) The size of the volume in GiB.
* `snapshot_id` - (String) The ID of the snapshot the volume is based on.
* `tags` - (Map of tags) Key-value map of tags assigned to the volume.
* `throughput` - (Int) The throughput that the volume supports in MiB/s.

> * `volume_id` - (String) he ID of the volume ID.

Reviewer: ```suggestion
* `volume_id` - (String) The ID of the volume.
```

In addition to all arguments above, the following attributes are exported:

* `arn` - The Amazon Resource Name (ARN) of the volume.
* `availability_zone` - The availability zone where the EBS volume will be located.
* `id` - The ID of the volume.
    * _Example:_ `vol-12345678`
* `iops` - The amount of IOPS for the volume.
* `size` - The size of the volume in GiB.
* `snapshot_id` - The ID of the snapshot the EBS volume is based on.
* `tags` - Map of tags assigned to the volume.
* `throughput` - The throughput that the volume supports, in MiB/s.

> * `volume_id` - The volume ID.

Reviewer 2: здесь тогда тоже
The ID of the volume

* `id` - The volume ID (e.g., vol-12345678).
* `volume_id` - The volume ID (e.g., vol-12345678).
* `arn` - The Amazon Resource Name (ARN) of the volume.
* `availability_zone` - The availability zone where the EBS volume will be located.
* `iops` - The amount of IOPS for the volume.
* `size` - The size of the volume in GiB.

> * `snapshot_id` - The snapshot_id the EBS volume is based off.

Reviewer 2: здесь должно быть словесно описание
```suggestion
* `snapshot_id` - The ID of the snapshot the EBS volume is based on.
```

* `volume_type` - The type of EBS volume.

> * `tags` - Map of tags assigned to the volume.

Reviewer 2: д.б. перед volume
